docs/getting-started.adoc

```diff
@@ -0,0 +1,196 @@
+= Getting Started
+
+This document walks you through the development and deployment of a Node.js application to an OpenShift cluster using `odo`. It guides you on how to push new changes to your application and on other useful features of `odo`.
+
+== Developing and deploying an application to an OpenShift cluster
+
+. link:/docs/installation.md[Install the `odo` binary].
+. Log into an OpenShift cluster:
+
+* If you lack a local development cluster, we recommend using
+https://docs.openshift.org/latest/minishift/getting-started/installing.html[Minishift]
+to deploy a development OpenShift cluster.
++
+.. Run Minishift:
++
+[source,bash]
+----
+$ minishift start
+----
+.. Log into the OpenShift cluster:
++
+[source,bash]
+----
+$ odo login -u developer -p developer
+----
++
+NOTE: In order to make full use of `odo` functionality, it is recommended to enable the OpenShift https://docs.openshift.com/container-platform/3.11/architecture/service_catalog/index.html[service catalog]. Use `minishift` 1.30 or latest to enable this: `sh   $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"`
```

Comment:
Note is printed inside a box, is this desired ? and command should be on the new line

Reply:
@surajnarwade fixed the command part of it.
As for the note in a box, that's as per standard asciidoc format used by the entire CCS, using this will keep it on Par with conventions followed in OpenShift docs.